openpathsampling/analysis/tis_analysis.py

```diff
@@ -65,27 +60,22 @@ def all_movers(self):
             all_movers += self.movers[movetype]
         return all_movers
 
-    def _assign_sample(self, sample):
-        if sample.ensemble in self.all_ensembles:
-            try:
-                self._samples_by_ensemble[sample.ensemble].append(sample)
-            except KeyError:
-                self._samples_by_ensemble[sample.ensemble] = [sample]
-            try: 
-                self._samples_by_id[sample.replica].append(sample)
-            except KeyError:
-                self._samples_by_id[sample.replica] = [sample]
+    # TODO: @dwhs can this function be removed?
+    # def _assign_sample(self, sample):
+    #     if sample.ensemble in self.all_ensembles:
+    #         try:
+    #             self._samples_by_ensemble[sample.ensemble].append(sample)
+    #         except KeyError:
+    #             self._samples_by_ensemble[sample.ensemble] = [sample]
+    #         try:
+    #             self._samples_by_id[sample.replica].append(sample)
+    #         except KeyError:
+    #             self._samples_by_id[sample.replica] = [sample]
```

Comment:
My IDE says the function is not used anywhere. So I was wondering if this is legacy that can safely be removed or is it something that you would like to keep. I commented it out to see, if the code breaks.